<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Particles</class>
 <widget class="QDialog" name="Particles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Particles</string>
  </property>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Min X:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="minX">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxX">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Min Y:</string>
   </property>
  </widget>
  <widget class="ImprovedLineEdit" name="velocity">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="colorView2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="minZ">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="update">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxY">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="color2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteAll">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete All</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="minY">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Z:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Max X:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetAll">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset All</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Min Z:</string>
   </property>
  </widget>
  <widget class="ImprovedLineEdit" name="gridSpacing">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="colorView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="ImprovedLineEdit" name="injectionPeriod">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxZ">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="particlesNew">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="color">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="ImprovedLineEdit" name="diffusionEffect">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="gridSpacingLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Grid Spacing:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="particlesDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Diffusion:</string>
   </property>
  </widget>
  <widget class="QLabel" name="injectionPeriodLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Injection Period:</string>
   </property>
  </widget>
  <widget class="QListView" name="particles">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>101</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="velocityLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Velocity:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="particlesType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>102</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Particles</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="passiveScalarType">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Passive Scalar</string>
   </property>
  </widget>
  <widget class="ImprovedLineEdit" name="injectionTime">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="injectionPeriodLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Injection Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>420</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="passiveScalarMomentPropagation">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>440</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>passive scalar</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>moment propagation</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Initial Concentration:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>470</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fixed Concentration:</string>
   </property>
  </widget>
  <widget class="ImprovedLineEdit" name="initialConcentration">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="ImprovedLineEdit" name="fixedConcentration">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>490</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImprovedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>improvedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>particlesType</tabstop>
  <tabstop>passiveScalarType</tabstop>
  <tabstop>particlesNew</tabstop>
  <tabstop>minX</tabstop>
  <tabstop>maxX</tabstop>
  <tabstop>minY</tabstop>
  <tabstop>maxY</tabstop>
  <tabstop>minZ</tabstop>
  <tabstop>maxZ</tabstop>
  <tabstop>color</tabstop>
  <tabstop>color2</tabstop>
  <tabstop>colorView</tabstop>
  <tabstop>colorView2</tabstop>
  <tabstop>update</tabstop>
  <tabstop>particles</tabstop>
  <tabstop>reset</tabstop>
  <tabstop>particlesDelete</tabstop>
  <tabstop>resetAll</tabstop>
  <tabstop>deleteAll</tabstop>
  <tabstop>gridSpacing</tabstop>
  <tabstop>diffusionEffect</tabstop>
  <tabstop>injectionPeriod</tabstop>
  <tabstop>velocity</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
